Makefile

```diff
@@ -24,7 +27,8 @@ test: build docs
 	python setup.py test pep8 pyflakes
 
 integration: build
-	PYFLAKES_NODOCTEST=True python setup.py integration pep8 pyflakes
+	INTEGRATION_TESTS=$(INTEGRATION_TESTS) PYFLAKES_NODOCTEST=True \
+		python setup.py integration pep8 pyflakes
```

Comment:
If you run nosetest directly you can specify the integration flag plus the particular directory which you want to invoke. Much simpler than having to tag each test with a unique name